src/Tizen.NUI.Components/Controls/Scrollbar.cs

```diff
@@ -413,6 +459,24 @@ public void SetCurrentValue(int currentValue, bool enableAnimation = true)
             CurrentValue = currentValue;
         }
 
+        /// <summary>
+        /// Apply style to ScrollBar.
+        /// </summary>
+        /// <param name="viewStyle">The style to apply.</param>
+        /// <since_tizen> 8 </since_tizen>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public override void ApplyStyle(ViewStyle viewStyle)
```

Comment:
Hello, 
please remove <since_tizen> tag, if it is hidden API. (tagged with [Editor ... Never] )

Reply:
ok